content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,330 @@
+        - name: data
+          type_:
+            columns:
+              - sepal length (cm)
+              - sepal width (cm)
+              - petal length (cm)
+              - petal width (cm)
+            dtypes:
+              - float64
+              - float64
+              - float64
+              - float64
+            index_cols: []
+            type: dataframe
+      name: predict_proba
+      returns:
+        dtype: float64
+        shape:
+          - null
+          - 3
+        type: ndarray
+  type: sklearn
+object_type: model
+requirements:
+  - module: sklearn
+    version: 1.0.2
+  - module: pandas
+    version: 1.4.1
+  - module: numpy
+    version: 1.22.3
+```
+
+To make ML model development Git-native, MLEM can work with DVC to manage
+versions of a model stored remotely in the cloud. Committing both model
+metainformation (`mlem-model.mlem`) and pointer to the model binary
+(`mlem-model.dvc` or `dvc.lock` if you train it in DVC pipeline) to Git allows
+you to enable GitFlow and other Software Engineering best practices like GitOps.
+
+<!-- If you train your model with
+DVC pipelines, DVC will handle this for you when you run a pipeline:
+
+```yaml
+# dvc.yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    outs:
+      - mlem-model
+      - mlem-model.mlem:
+          cache: false
+```
+
+And if you don't use DVC pipelines, you can just track your model binary with
+DVC:
+
+```shell
+$ dvc add mlem-model
+Now tracking `mlem-model` with DVC.
+Run `git add mlem-model.dvc` and then `git commit`.
+``` -->
+
+# Running your models anywhere
+
+The main goal of MLEM is to provide you a single tool that enables any kind of
+model productionization scenarios. For MLEM, there are three main groups of
+those scenarios:
+
+- **Use** a model direcly with MLEM.
+- **Export** a model to a format that can be used by other tools.
+- **Deploy** a model to a production environment or cloud provider.
+
+The first one allows you to import your model into a Python runtime, run predict
+against some dataset directly in the command line, or serve the model with MLEM
+from your CLI.
+
+```python
+$ python
+>>> import mlem
+>>> model = mlem.api.load("mlem-model")
+>>> model.predict(test)
+[[0.4, 0.3, 0.3], [0.2, 0.5, 0.3]]
+```
+
+```shell
+$ mlem apply mlem-model test.csv
+[[0.4, 0.3, 0.3], [0.2, 0.5, 0.3]]
+```
+
+```shell
+$ mlem serve ml-model fastapi
+⏳️ Loading model from ml-model.mlem
+Starting fastapi server...
+💅 Adding route for /predict
+💅 Adding route for /predict_proba
+Checkout openapi docs at <http://0.0.0.0:8080/docs>
+INFO:     Started server process [5750]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
+```
+
+The second one allows you to package your models as a Python package, build a
+Docker Image, or export it as some special format (like `.onnx` which is coming
+soon).
+
+```shell
```

Comment:
@iterative/websites do we have syntax highlighters ready for MLEM?

Reply:
@yathomasi, we can't use ```cli``` highlighter here also yet, right?